site/contents/write_ups/2021/15-11-21_writeup.md

```diff
@@ -0,0 +1,85 @@
+
+Some people have the viewpoint that anyone who does not trust scientists/doctors/etc. is a conspiracy theorist or simply not worth talking to. This can be extremely polarising. It is also problematic to assume that everyone from a particular group or progression has the same views.
+
+We were unable to come to a consesus as to whether this phenomenom is getting better or worse. Intellectual superiority in universities reinforces this mindset, creating a bubble of knowledge where you become ignorance of other people's (or indeed your own) ignorance. It is important that we can admit that we are wrong but the culture in academia is not very conducive when it comes to encouraging this. There is very public evidence of people being humiliated by colleagues for making relatively minor mistakes. Issues of irreproducibility in scientific results can also undermind trust.
+
+As the current Covid-19 pandemic has bought scientific advice to the forefront of how we live our lives. Perhaps as a result of this, the public trust statisticians less, same as the weatherman. There is a difference between what people describe as a data scientist versus a data analyst. 
+
+Profession is a big word and maybe it is not yet fair to claim that there is a "data science" profession. "Any claim I make I need to back it up" is a key for scientific disciplines, but not always possible. Assessability is not always available as some data is hidden.
+
+We need to improve public statistical literacy and understanding how quantitative circles back to qualitative. In the future we need to make sure we don't gatekeep based on education. Value judgements are important, feelings versus fact and what is empirical. Science is not neutral, scientific questions and discoveries are not neutral. Any human endeavour has confirmation bias present.
+
+## What are your thoughts on the use of statistics in the media, and how this contributes to trust/mistrust in scientific evidence by the public?
+
+People are not always trained to interpret graphs or statistics. Even when we are trained it can be difficult. Language can be very opaque - researchers don't always put in the effort to communicate complex things to the public, which perhaps links with academics assuming the public are stupid?
+
+Social media can be used for good. People are using it to ask the public questions and build trust by being available to discuss feedback. On the other hand social media is an echo chamber of people engaging with the content that they are interested in, and agree with. 
+
+We need to remember that science is not designed to be ingested paper by paper. It's supposed to be a field where messages are shared based on collective evidence. Preprints getting into the news before being peer reviewed set people's expectations before 
+
+It's easier when we already trust someone's opinion to believe what they say subsequently.
```

Comment:
Maybe move this to line 29 as end of that paragraph?

```suggestion
Building trust with the public is especially important because it's easier when we already trust someone's opinion to believe what they say subsequently.
```